community/modules/scripts/ramble-setup/variables.tf

```diff
@@ -15,43 +15,47 @@
  */
 
 variable "install_dir" {
-  description = "Destination directory of installation of Ramble"
+  description = "Destination directory of installation of Ramble."
   default     = "/apps/ramble"
   type        = string
 }
 
 variable "ramble_url" {
-  description = "URL for Ramble repository to clone"
+  description = "URL for Ramble repository to clone."
   default     = "https://github.com/GoogleCloudPlatform/ramble"
   type        = string
 }
 
 variable "ramble_ref" {
-  description = "Git ref to checkout for Ramble"
+  description = "Git ref to checkout for Ramble."
   default     = "develop"
   type        = string
 }
 
 variable "log_file" {
-  description = "Log file to write output from ramble setup steps into"
+  description = "Log file to write output from ramble setup steps into."
   default     = "/var/log/ramble-setup.log"
   type        = string
 }
 
 variable "chown_owner" {
-  description = "Owner to chown the Ramble clone to"
-  default     = "root"
+  description = "Owner to chown the Ramble clone to. Default will not modify the clone."
+  default     = null
   type        = string
 }
 
 variable "chgrp_group" {
-  description = "Group to chgrp the Ramble clone to"
-  default     = "root"
+  description = "Group to chgrp the Ramble clone to. Default will not modify the clone."
+  default     = null
   type        = string
 }
 
 variable "chmod_mode" {
-  description = "Mode to chmod the Ramble clone to."
-  default     = ""
+  description = <<EOT
+    Mode to chmod the Ramble clone to. Defaults to null (i.e. do not modify).
+    For usage information see:
+    https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-mode
+EOT
```

Comment:
nit: if you use `<<-EOT` instead you should be able to indent the final `EOT` and the output in docs will look cleaner.

Reply:
Done